tests/features/support/hooks.js

```diff
@@ -0,0 +1,64 @@
+var {defineSupportCode} = require('cucumber');

```

Comment:
__[jshint]__
* 'destructuring binding' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).

<!-- Comment by [SideCI](https://sideci.com) -->